erpnext/stock/doctype/delivery_note/delivery_note.py

```diff
@@ -195,6 +195,8 @@ def update_current_stock(self):
 
 	def on_submit(self):
 		self.validate_packed_qty()
+		if frappe.db.get_single_value("Accounts Settings", "auto_create_invoice_on_delivery_note") == "Submit":
+			self.make_sales_invoice_for_delivery()
```

Comment:
This happened on `before_submit` in https://github.com/DigiThinkIT/bloomstack_core/pull/417.

Should this happen on `before_submit` instead?

Reply:
Okay